docs/markdown/uix-core.extension.md

```diff
@@ -20,6 +20,7 @@ This interface is likely to expand. As the metadata from the extension registry
 
 |  Property | Modifiers | Type | Description |
 |  --- | --- | --- | --- |
+|  [configuration?](./uix-core.extension.configuration.md) |  | Record&lt;string, unknown&gt; | _(Optional)_ An extension configuration |
```

Comment:
```suggestion
|  [configuration?](./uix-core.extension.configuration.md) |  | Record&lt;string, unknown&gt; | _(Optional)_ Configuration of the extension for a specific environment |
```

Reply:
@lenaorobei , this file was auto-generated from code annotations, if we decide to update that definition, I'll change the related annotations